recipe/meta.yaml

```diff
@@ -9,8 +9,14 @@ source:
   md5: 0febfafd14330c9dcaa40de2d82d40ad
 
 build:
-  number: 1000
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 1001
+  script:
+    ###########################################################
+    # Delete broken symlink, which causes build to fail.      #
+    # xref: https://github.com/testing-cabal/mock/issues/446  #
+    ###########################################################
+    - del /q docs\\changelog.txt  # [win and py27]
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
Apparently `pip install` will [fail on Windows Python 2.7]( https://ci.appveyor.com/project/conda-forge/mock-feedstock/builds/22603672/job/iy9k6vq2y17loj3h ) because of a broken link at `docs\changelog.txt`. This removes the file as a workaround, which [fixes it]( https://ci.appveyor.com/project/conda-forge/mock-feedstock/builds/22604551/job/iacx20bp3btvo6hs ). Have raised upstream issue ( https://github.com/testing-cabal/mock/issues/446 ).